packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -755,4 +775,108 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/patient/ops/coverage-assessment
+ *
+ * return the coverage
+ * @param req.query.cxId The customer ID.
+ * @param req.params.id The patient ID.
+ * @param req.query.facilityId The facility ID for running the coverage assessment.
+ * @param req.query.dryrun Whether to simply validate or run the assessment.
+ * @return A patient.
+ *
+ */
+router.post(
+  "/ops/coverage-assessment",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const facilityId = getFrom("query").orFail("facilityId", req);
+    const dryrun = getFromQueryAsBoolean("dryrun", req);
+    const payload = coverageAssessmentSchema.parse(req.body);
+
+    if (dryrun) return res.status(status.OK);
+
+    const facility = await getFacilityOrFail({ cxId, id: facilityId });
+    const patientCreates: PatientCreateCmd[] = payload.patients.map(patient => {
+      const phone1 = patient.phone1 ? normalizePhoneNumberStrict(patient.phone1) : undefined;
```

Comment:
Wouldn't it be better to have these on the cmd, so the route is very thin, just getting IDs/params from the request and validating the zod schema?

Could pass `dryrun` to the cmd, we do this in other cases too.

Reply:
Reworking the cmd to accept a flatter structure (as well as other array / object types) I agree with, but I'm not sure it should be apart of this PR and likely yields max value if we're willing to flatten the patient more generally, which is a larger change.

The `contact` array is generally a pain and I think we end up having to transform it almost everywhere we touch it - it's just an overly complicated typed attribute for what is really two string lists: phone numbers and emails...